### PR TITLE
Finishing auto generating game server.

### PR DESCRIPTION
**Pull request by Rain36522**

### CODE REVIEW

File: GameServer/game/game.py [hunk 1]
```diff
@@ -30,21 +31,47 @@
...
     "user4" : ""
 }
 
+def putDatagameSettings(data, settings):
+    if data.get("ballwidth"):
+        settings["ballwidth"] = data["ballwidth"]
+    if data.get("planksize"):
+        settings["planksize"] = data["planksize"]
+    if data.get("Speed"):
+        settings["Speed"] = data["Speed"]
```
Comment (DavePie) [inline GameServer/game/game.py:46]: Put into an array and do a loop

File: GameServer/game/game.py [hunk 2]
```diff
@@ -30,21 +31,47 @@
...
+    if data.get("user3"):
+        settings["user3"] = data["user3"]
+    if data.get("user4"):
+        settings["user4"] = data["user4"]
+    return settings
+
+
 # Exemple d'utilisation du client WebSocket avec asyncio
 if __name__ == "__main__":
+    print("\033[31m", file=sys.stderr)
```
Comment (DavePie) [inline GameServer/game/game.py:66]: Remove

File: TranServer/game/urls.py [hunk 3]
```diff
@@ -3,6 +3,6 @@
 from .views import newGame
 
 urlpatterns = [
-    
-    path('newGame/', newGame, name='newGame')
+    path('newGame/', newGame.as_view(), name='newGame'),
+    # path('api/chat/<str:game_id>/', name='newgame_api'),
```
Comment (DavePie) [inline TranServer/game/urls.py:7]: remove

**Review by DavePie — CHANGES_REQUESTED** [review]

File: GameServer/game/game.py (hunk 2 above)
Comment (Rain36522) [reply GameServer/game/game.py:66]: Do in loop

**Review by DavePie — APPROVED** [review]
Looking good!

**Pull request merged**